pkg/cluster/internal/kubeadm/config.go

```diff
@@ -134,6 +134,20 @@ nodeRegistration:
     node-ip: "{{ .NodeAddress }}"
 networking:
   podSubnet: "{{ .PodSubnet }}"
+# we need this to allow dynamic PVs to work properly
+apiServerExtraArgs:
+  "feature-gates": "DynamicProvisioningScheduling=true"
```

Comment:
the docs says it was superseded by `VolumeScheduling`
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
this config is for kubernetes v1.11 :upside_down_face:

Reply:
it was deprecated in 1.12

Reply:
ahh, but only after 1.12 

> DynamicProvisioningScheduling: Extend the default scheduler to be aware of volume topology and handle PV provisioning. This feature is superseded by the VolumeScheduling feature completely in v1.12.